src/swish/io.ss

```diff
@@ -853,6 +854,14 @@
          os-pid)]
        [(,who . ,errno) (io-error path who errno)])))
 
+  (define (spawn-os-process-detached path args)
+    (unless (and (list? args) (for-all string? args))
+      (bad-arg 'spawn-os-process-detached args))
+    (with-interrupts-disabled
```

Comment:
There's no need to disable interrupts here because there are no interruptable parts inside.

Reply:
Good catch!